.github/workflows/cli.yml

```diff
@@ -28,6 +28,11 @@ jobs:
         with:
           go-version: ${{ matrix.go }}
 
+      - name: Use Node.js 10.x
```

Comment:
Question: Any reason to prefer `10.x`? Latest LTS version is `12`. 

https://nodejs.org/en/about/releases/

Reply:
I am using 10.x on my system where it was working so I used that. Let me try to change it to 12